components/composition/Wrap/Wrap.css

```diff
@@ -0,0 +1,67 @@
+/* stylelint-disable selector-no-qualifying-type */
```

Comment:
I think we can use nesting to not have to repeat diamond-wrap? https://developer.mozilla.org/en-US/docs/Web/CSS/Nesting_selector

Reply:
Not sure CSS nesting is ready for prod yet? https://caniuse.com/css-nesting

Reply:
We could polyfill it via postCSS, I was kinda trying to stay away from a processor until I had to

Reply:
It looks fine on can i use it can be processed by consuming sites anyways

Reply:
No support on Samsung internet at 3.36% of UK users is the one that stands out to me

Reply:
But if it's preprocessable, maybe we should just do it